Дз сделано, но не на мой максимум.

VaolEr opened the pull request:
Работают все функции показанные на уроке, но:
из за отстутсвия времени по рабочим причинам и обширности задачи
1. Не решена задача с таймером на 120 секунд;
2. Не доведён до ума интерфейс;
3. В получении сообщений нарушена последовательность sender, message.

VaolEr committed:
Работают все функции показанные на уроке, но:
из за отстутсвия времени по рабочим причинам и обширности задачи
1. Не решена задача с таймером на 120 секунд;
2. Не доведён до ума интерфейс;
3. В получении сообщений нарушена последовательность sender, message.

OKrylov reviewed (changes requested):
буду ждать реализацию с таймером. Если будут вопросы - пишите

VaolEr committed:
Работают все функции показанные на уроке, но:
из за отстутсвия времени по рабочим причинам и обширности задачи
1. Решена задача с таймером на 120 секунд;
2. Доведён до ума интерфейс;
3. В получении исправлена последовательность sender, message.

VaolEr commented:
Will be new PR with correct project.
